xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -43,16 +43,17 @@ func init() {
 	}
 }
 
-// RegisterForTesting registers the RBAC HTTP Filter for testing purposes, regardless
-// of the RBAC environment variable. This is needed because there is no way to set the RBAC
-// environment variable to true in a test before init() in this package is run.
+// RegisterForTesting registers the RBAC HTTP Filter for testing purposes,
+// regardless of the RBAC environment variable. This is needed because there is
+// no way to set the RBAC environment variable to true in a test before init()
+// in this package is run.
```

Comment:
Same here about being able to remove this once the env var is gone. If so, please add TODO.